apiserver/billing/odoostorage/odoostorage.go

```diff
@@ -7,19 +7,28 @@ import (
 	billingv1 "github.com/appuio/control-api/apis/billing/v1"
 	"github.com/appuio/control-api/apiserver/billing/odoostorage/odoo"
 	"github.com/appuio/control-api/apiserver/billing/odoostorage/odoo/fake"
+	"github.com/appuio/control-api/apiserver/billing/odoostorage/odoo/odoo8"
 )
 
-// New returns a new storage provider for Organizations
-func New() Storage {
+// NewFakeStorage returns a new storage provider for BillingEntities
+func NewFakeStorage(metadataSupport bool) Storage {
```

Comment:
Maybe I'm missing something, but what's the point of the `metadataSupport` function parameter, if we don't use it?

Reply:
🤦‍♀️ should have been used 2 lines further down.